src/main/webapp/js/antidote.js

```diff
@@ -42,10 +42,15 @@ function runSnippetInTab(tabName, snippetIndex) {
     // Select tab
     $('.nav-tabs a[href="#' + tabName + '"]').tab('show')
 
+    if (typeof snippetIndex == 'number') {
+        var snippetText = document.getElementById('labGuide').getElementsByTagName('pre')[parseInt(snippetIndex)].innerText;
+    } else if (typeof snippetIndex == 'object') {
```

Comment:
```suggestion
    } else {
```

Let's just catch all non-number cases here. If there's an issue accessing the object fields, that will get popped up to the javascript console.

Reply:
Applied the change according to your suggestion, and updated the CHANGELOG as well.